src/main/scala/io/sdkman/changelogs/SparkMigrations.scala

```diff
@@ -55,11 +55,18 @@ class SparkMigrations {
 
   @ChangeSet(order = "007", id = "007-add_spark_2.4.x", author = "marc0der")
   def migration007(implicit db: MongoDatabase) = {
-    Seq("2.4.1", "2.4.2", "2.4.3", "2.4.4").foreach { version =>
+    Seq("2.4.1", "2.4.2", "2.4.3").foreach { version =>
       Version("spark", version, s"https://archive.apache.org/dist/spark/spark-$version/spark-$version-bin-hadoop2.7.tgz")
         .validate()
         .insert()
     }
-    setCandidateDefault("spark", "2.4.4")
+    setCandidateDefault("spark", "2.4.3")
+  }
+
+  @ChangeSet(order = "008", id = "008-add_spark_2.4.4", author = "marc0der")
+  def migration008(implicit db: MongoDatabase) = {
+    Version("spark", "2.4.4", "https://archive.apache.org/dist/spark/spark-2.4.4/spark-2.4.4-bin-hadoop2.7.tgz")
+      .validate()
+      .insert()
```

Comment:
After inserting, you can also call the `.asCandidateDefault()` to make this the default version.

Reply:
Done.